.github/workflows/ci_extended.yaml

```diff
@@ -106,4 +100,4 @@ jobs:
 
     - name: Run tests
       working-directory: build
-      run: ctest -R ^xtest$ --output-on-failure
+      run: ctest -R ^xtest$ --verbose
```

Comment:
```suggestion
      run: ctest -R ^xtest$ --output-on-failure
```

Reply:
Yeah I was playing around with that to see if I could get a better output. Didn't work